File: week03/week03-4.py
# week03-4.py
# 學習計畫第3題 大海撈針 在haystack 裡找needle
# LeetCode 28.Find the Index of the First Occurrence in a  String
#在 Python 只要一行 a.find(b) 在字串a裡找到字串b
class Solution:
    def strStr(self, a: str, b: str) -> int:
        return a.find(b)

File: week03/week03-4.cpp
// week03-4.cpp 學習計畫第3題 大海撈針 在haystack 裡找needle
//LeetCode 28.Find the Index of the First Occurrence in a  String
class Solution {
public:
    int strStr(string haystack, string needle) {
        ///https://cplusplus.com/reference/string/string/find/#google_vignette
        return haystack.find(needle);
    }
};